packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -213,7 +213,7 @@ const ModalElement = styled(AnimatedView)<TransitionElementProps>`
   top: 50%;
   left: 50%;
   height: ${props => (props.height ? props.height : "auto")};
-  max-height: calc(100% - 80px);
+  max-height: calc(100vh - 80px);
```

Comment:
Ah, was thinking that this seemed  kinda weird no a large screen but checked zeplin and this is the spec 👍